android/app/src/main/res/layout/large_widget_layout.xml

```diff
@@ -0,0 +1,304 @@
+                android:textColor="@android:color/black"
+                android:textSize="15sp"
+                tools:text="Suomi" />
+
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/hourForecastRowLayout"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="6dp"
+                android:layout_marginBottom="6dp"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/timeTextView0"
+                    style="@style/time"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal|top"
+                    tools:text="13:00" />
+
+                <ImageView
+                    android:id="@+id/weatherIconImageView0"
+                    android:layout_width="36dp"
+                    android:layout_height="36dp"
+                    android:layout_gravity="center_horizontal"
+                    android:contentDescription="TODO"
+                    tools:src="@drawable/s32_light" />
+
+                <TextView
+                    android:id="@+id/temperatureTextView0"
+                    style="@style/temperatureSmall"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal"
+                    tools:text="+12°" />
+
+            </LinearLayout>
+
+            <!-- Note: 'View' cannot be used with widgets, therefore ImageView here -->
+            <ImageView
+                style="@style/verticalLine"
+                android:layout_width="1dp"
+                android:layout_height="match_parent"/>
+
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="6dp"
+                android:layout_marginBottom="6dp"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/timeTextView1"
+                    style="@style/time"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal|top"
+                    tools:text="13:00" />
+
+                <ImageView
+                    android:id="@+id/weatherIconImageView1"
+                    android:layout_width="36dp"
+                    android:layout_height="36dp"
+                    android:layout_gravity="center_horizontal"
+                    android:contentDescription="TODO"
```

Comment:
Texts for this one can be found: https://github.com/fmidev/weather-app/blob/9fdbf9cac7caf6103ece3dcf208f38a8e9e9328c/i18n/fi.json#L455

Create new ticket for accessibility labels if you don't implement it in this ticket.

Reply:
New ticket created:
https://github.com/fmidev/weather-app/issues/714